lib/BC/InstructionLifter.cpp

```diff
@@ -290,27 +291,28 @@ InstructionLifter::LoadRegAddress(llvm::BasicBlock *block,
 
     reg_ptr_it->second = reg_ptr;
     return reg_ptr;
+  }
 
   // Try to find it as a global variable.
-  } else if (auto gvar = module->getGlobalVariable(reg_name)) {
+  if (auto gvar = module->getGlobalVariable(reg_name)) {
     return gvar;
+  }
 
   // Invent a fake one and keep going.
-  } else {
-    std::stringstream unk_var;
-    unk_var << "__remill_unknown_register_" << reg_name;
-    auto unk_var_name = unk_var.str();
-    auto var = module->getGlobalVariable(unk_var_name);
-    if (!var) {
-      LOG(ERROR)
-          << "Could not locate variable or register " << reg_name_;
-
-      var = new llvm::GlobalVariable(
-          *module, impl->word_type, false, llvm::GlobalValue::ExternalLinkage,
-          llvm::UndefValue::get(impl->word_type), unk_var_name);
-    }
+  std::stringstream unk_var;
+  unk_var << "__remill_unknown_register_" << reg_name;
+  auto unk_var_name = unk_var.str();
+  auto var = module->getGlobalVariable(unk_var_name);
+  if (var) {
```

Comment:
`if( auto var = ... )` or is there any reason for it to have a it's own line (and therefore longer lifespan?)